debugger/ptraceLib/contract.h

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#ifndef _OE_CONTRACT_H
```

Comment:
Can we rename this defined to just _CONTRACT_H or perhaps _PTRACELIB_CONTRACT_H?
The define should match the filename for upstream contribution.

Reply:
Fixed.